### PR TITLE
Simplify the policy

### PR DESCRIPTION
**Pull request by kylewillmon**
This patch simplifies our default policy and combines it with an extra example that uses domain-specific thresholds.

### CODE REVIEW

File: per_domain.rego [hunk 1]
```diff
@@ -0,0 +1,21 @@
+package policy
+
+import future.keywords.in
+import data.phylum.domain
+import data.phylum.level
+
+issue["risk level cannot exceed medium"] {
```
Comment (matt-phylum) [inline per_domain.rego]: What do you think about this variation?

```rego
issue contains "risk level cannot exceed medium" if {
    data.issue.domain in {domain.AUTHOR, domain.ENGINEERING, domain.VULNERABILITY}
    data.issue.severity > level.MEDIUM
}
```

It requires `import future.keywords.contains` `import future.keywords.if` (Or maybe only one of them. One might include the other.). Doing it this way might read better to users that are new to rego.

**Review by matt-phylum — APPROVED** [review]
Two things:

1. We need to make sure users will understand what to do. @furi0us333 requested that I keep the thresholds broken out like that because that way if a user is used to the sliders and they get to here it's obvious what they should do to get similar behavior to before.
2. This is not currently the source of default.rego (in fact it's a little different from the real default.rego even). The real version that is being used by the API lives in the private API repository. I think it would make sense if we were pulling it from here instead since we have this repository now. 👍

File: per_domain.rego (hunk 1 above)
Comment (kylewillmon) [reply per_domain.rego]: It turns out that `opa fmt` will add these for you if you just import the keywords. :+1:

**Review by matt-phylum — APPROVED** [review]

**Review by furi0us333 — APPROVED** [review]
Given the per domain example code is available, I am fine with this approach.

**Pull request merged**